.renvignore

```diff
@@ -1,5 +1,7 @@
 README.Rmd
 pipeline/00-ingest.R
-pipeline/06-export.R
-pipeline/07-api.R
-reports/
\ No newline at end of file
+pipeline/05-finalize.R
```

Comment:
Unfortunately it seems like having `renv` look at `finalize` also means it looks into the Quarto doc itself i.e. there's no way to ignore finalize while also ignoring the doc. I think it's fine since all the finalize dependencies are covered by prior steps and are unlikely to change.

Reply:
Fixed in 2cf6cebd53e154353c774a34f94a560c18be63e3!